project_internal_access_from_portal/__manifest__.py

```
@@ -0,0 +1,20 @@
+    "author": "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["project", "portal"],
+    "data": [
+        "security/ir.model.access.csv",
+        "security/portal_project_rules.xml",
+    ],
+    "demo": ["demo/demo_data.xml"],
+    "installable": True,
+    "application": False,
+    "auto_install": False,
+}
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)